spec/class.dd

```diff
@@ -184,16 +184,25 @@ $(SPEC_RUNNABLE_EXAMPLE_RUN
 ---
 class Foo { int x; long y; }
 
+static assert(Foo.tupleof[0].stringof == "x");
+static assert(typeof(Foo.tupleof).stringof == `(int, long)`);
```

Comment:
Assuming an exact result for `.stringof` is bad practice, since the string representation for a type or expression can vary per implementation.

Reply:
OK, updated.